sdk/servicebus/azure-messaging-servicebus/pom.xml

```diff
@@ -67,12 +72,6 @@
       <version>1.13.11</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
     </dependency>
 
-    <dependency>
-      <groupId>com.fasterxml.jackson.dataformat</groupId>
-      <artifactId>jackson-dataformat-xml</artifactId>
```

Comment:
This is a breaking change and I wonder what @srnagar and @JonathanGiles think about it.

In any case, we should at least mention it in the changelog under breaking changes.

Reply:
Yeah, I don't think we can remove a dependency as it's a breaking change. We should bump the major version if we have to do it.

Reply:
Would changing the scope to provided be sufficient for removing our own usage of Jackson Dataformat XML while not being break? Or, is a clean break preferred?

The point of azure-json and azure-xml is to begin the full removal of Jackson as a dependency from the SDKs. So, this is a good starting point for how we want to handle this as this will keep popping up.